src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2407,20 +2415,13 @@ class CreateFunctionStatementSegment(BaseSegment):
 
 
 @ansi_dialect.segment()
-class FunctionParameterGrammar(BaseSegment):
+class FunctionParameterListGrammar(BaseSegment):
```

Comment:
Renamed this to distinguish function parameter **list** from a single function **parameter**